modules/rules_go/metadata.json

```diff
@@ -8,7 +8,9 @@
         }
     ],
     "versions": [
-        "0.33.0"
```

Comment:
oops, something should have caught that 0.34 was dead code previously...

Reply:
Yes, we could definitely use more CI checks for this and other common classes of errors.

Reply:
I'll work on adding more presubmit checks soon